src/objects-table/ObjectsTable.js

```diff
@@ -177,20 +180,22 @@ class ObjectsTable extends React.Component {
         ev.stopPropagation();
 
         const { selection } = this.state;
-        const deletedIds = _.remove(selection, id => id === obj.id);
-        if (deletedIds.length === 0) selection.push(obj.id);
+        const found = selection.delete(obj.id);
+        if (!found) selection.add(obj.id);
```

Comment:
I think JS sets operations are mutable, right? We need immutable updates to play nice tiwh React. Maybe Array.from + append + new Set()?

Reply:
Not sure about this login of delete followed by an add to do a toggle. Anyway, it will be refactored when doing it immutable.

Reply:
The logic was, try to delete if it didn't exist create it. One of the less verbose ways to do a toggle.

But I'll redo with immutable operations.